.changeset/chatty-bugs-perform.md

```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": patch
+"@jpmorganchase/uitk-theme": patch
+---
+
+Content status add typography
```

Comment:
Sorry to be a pain but can we split the changesets for theme and lab out and make the theme one more specific? Also this is probably a minor change